src/Components/SmartComponents/Reports/Common/firstPagePDF.js

```diff
@@ -35,29 +37,49 @@ const firstPagePDF = ({ data, meta, filters, intl, isReportDynamic, reportData,
         }
     }
 
+    // eslint-disable-next-line no-console
+    console.log('test', filterStringSegments);
+    // eslint-disable-next-line no-console
+    console.log('filtersBuilder', filtersBuilder);
```

Comment:
I think you forgot something here :smile:

Reply:
oups. I disabled the console.log error and that's the results :(